### DISCUSSION TITLE
How to use adl64.exe for debugging?

### DISCUSSION THREAD

**Discussion by Adolio** [BowlerHatLLC/vscode-as3mxml#748] [Q&A] [answered]
Hello,

On Windows, is it possible to configure my project to use the `64-bit` version of `ADL` (`adl64.exe`)?

By default, even if my `application.xml` contains the `<architecture>64</architecture>` option, the `32-bit` version of `ADL` is used for debugging (everything is fine when packaging the app). Is this the intended behavior?

Thanks for your help,
Aurélien

**Comment by waveofthought-code** [comment, marked as answer]
in launch.json, add a `runtimeExecutable` property to your configurations, like this example:

```    
"configurations": [
        {
            "type": "swf",
            "request": "launch",
            "name": "Debug Local",
            "program": "your-application-xml.xml",
            "runtimeExecutable": "C:/path/to/AirSDK/bin/adl64.exe",
        }
```

**Reply by Adolio** [reply]
Nice, thanks!

I also managed to directly use the configured SDK with:

```
"runtimeExecutable": "${config:as3mxml.sdk.framework}/bin/adl64.exe"
```

**Reply by waveofthought-code** [reply]
Oh cool I just learned something too :D
